pybamm/logger.py

```diff
@@ -12,7 +17,5 @@
 
 # Create a custom logger
 logger = logging.getLogger(__name__)
+set_logging_level("WARNING")
```

Comment:
loks like the `test_logger` test needs updating as the default logger level has changed